classes/logged_in.php

```diff
@@ -1,8 +1,37 @@
 <?php
   class logged_in 
   {
+    public $userrole;
+
     public function __construct() 
```

Comment:
The construct function from php is amazing on line six, you created the PHP function and that's really technical and cool.  We love your contribution and hope that you could use this function more frequently! The function is made on a user-level side and that's fine, line 4 on the public userrole makes it perfect!